main.go

```diff
@@ -40,25 +42,47 @@ func main() {
 	o.Path("/login-complete").Methods("GET").HandlerFunc(handler.LoginComplete)
 
 	a := r.PathPrefix("/api").Subrouter()
-	a.Use(authenticator.Middleware)
+	// a.Use(authenticator.Middleware)
 
 	s := a.PathPrefix("/elections").Subrouter()
-	s.Methods("GET").HandlerFunc(handler.GetElectionIds)
-	s.Methods("POST").HandlerFunc(handler.CreateElection)
+	s.Methods("GET").HandlerFunc(handler.GetElections)
+	s.Path("/create").Methods("POST").HandlerFunc(handler.CreateElection)
 
 	s = a.PathPrefix("/election/{electionId}").Subrouter()
+	s.Methods("GET").HandlerFunc(handler.GetElectionInfo)
+	s.Methods("POST").HandlerFunc(handler.ModifyElection)
 	s.Path("/publish").Methods("PUT").HandlerFunc(handler.PublishElection)
 	s.Path("/close").Methods("PUT").HandlerFunc(handler.CloseElection)
 	s.Path("/vote").Methods("POST").HandlerFunc(handler.CastVote)
-	s.Path("/votes").Methods("GET").HandlerFunc(handler.GetVotes)
-	s.Path("/voters").Methods("GET").HandlerFunc(handler.GetEligibleVoters)
-	s.Path("/voters").Methods("PUT").HandlerFunc(handler.AddEligibleVoters)
+	s.Path("/votes").Methods("GET").HandlerFunc(handler.GetElectionVotes)
+	s.Path("/votes/count").Methods("GET").HandlerFunc(handler.CountElectionVotes)
+
+	s = a.PathPrefix("/candidates").Subrouter()
+	s.Methods("GET").HandlerFunc(handler.GetAllCandidates)
+	s.Path("/create").HandlerFunc(handler.CreateCandidate)
+
+	s = a.PathPrefix("/candidate/{candidateID}").Subrouter()
+	s.Methods("GET").HandlerFunc(handler.GetCandidate)
+	s.Methods("POST").HandlerFunc(handler.ModifyCandidate)
+	// s.Methods("DELETE").HandlerFunc(handler.DeleteCandidate)
+
+	s = a.PathPrefix("/voters").Subrouter()
+	s.Methods("GET").HandlerFunc(handler.GetValidVoters)
+	s.Path("/add").Methods("PUT").HandlerFunc(handler.AddValidVoters)
+	s.Path("/remove").Methods("PUT").HandlerFunc(handler.RemoveValidVoters)
+
+	s = a.PathPrefix("/history").Subrouter()
+	s.Methods("GET").HandlerFunc(handler.GetLogs)
+
+	s = a.PathPrefix("/cleardb").Subrouter()
+	s.Methods("PUT").HandlerFunc(handler.NukeSystem)
```

Comment:
Isn't clearing resources generally done by `DELETE` method?

Reply:
I've thought a little about this, and I think I will rename this endpoint to `/reset-system` instead, since that captures the intent of the endpoint much better than `/cleardb`. I think `PUT` makes more sense for that name?